docs/Getting-Started/Create-bootable-media/create-bootable-usb.md

```diff
@@ -0,0 +1,36 @@
+# Create Pop!_OS Installation Media
+
+Pop!_OS can be run from a USB drive in a live environment. You can use the live USB to install Pop!_OS permenantly on a system, demo Pop!_OS without installing, or use the drive as a system recovery tool.
+
+---
+## Preparation
+
+Creating a bootable Pop!_OS USB drive requires three components: 
+
+- A USB flash drive. The flash drive must have at least 8 GB of free space. All data on the flash drive will be lost during the creation process.
+
+- The Pop!_OS.iso image: [Download Pop!_OS at this link.](https://pop.system76.com/)
+
+>**Note**: System76 offers their Pop!_OS.iso in 3 varieties: Intel/AMD, NVIDIA, and RAS PI. See [Choosing a Pop!_OS Image](create-bootable-usb.md#choose-a-pop_os-image).
+
+- A program to write the Pop!_OS.iso image to the flash drive. [Etcher](https://www.balena.io/etcher/) and [Popsicle](https://github.com/pop-os/popsicle) are popular options for Linux users, and [Rufus](https://rufus.ie/en/) is compatible with both Windows and MacOS.
+
+## Choose a Pop!_OS Image
```

Comment:
I don't like that we had an aside about this just two paragraphs ago, then we're rehashing it here. Maybe the aside's not necessary, or maybe that bullet point above should just link down here (and the download link can be here) instead of to the download page if you think the information is critical?

Reply:
I had added it there in case people don't read any further than the download link. Taking a second look, it doesn't serve much purpose and I also don't like adding a section link to content that is currently already visible on the page.